builtin/logical/pki/path_tidy.go

```diff
@@ -1459,6 +1496,21 @@ func (b *backend) pathConfigAutoTidyWrite(ctx context.Context, req *logical.Requ
 		}
 	}
 
+	if revokedSafetyBufferRaw, ok := d.GetOk("revoked_safety_buffer"); ok {
+		if config.RevokedSafetyBuffer == nil {
+			config.RevokedSafetyBuffer = new(time.Duration) // Allocate memory for pointer
+		}
+		*config.RevokedSafetyBuffer = time.Duration(revokedSafetyBufferRaw.(int)) * time.Second
```

Comment:
```suggestion
		revokedSafetyBuffer := time.Duration(revokedSafetyBufferRaw.(int)) * time.Second
		*config.RevokedSafetyBuffer = &revokedSafetyBuffer
```

Should be fewer lines and avoids a weird heap allocation. :-)

Reply:
Looks much better 👀